libs/designsystem/src/lib/kirby.module.ts

```diff
@@ -115,6 +116,7 @@ const exportedDeclarations = [
   DropdownComponent,
   InfiniteScrollDirective,
   LoadingOverlayComponent,
+  HighstockChartComponent,
```

Comment:
Consider moving to after ChartComponent for readability... 👀

Reply:
Done.

Reply:
Done